nemo_curator/utils/text_utils.py

```diff
@@ -23,16 +23,33 @@
 
 def get_word_splitter(language):
     language = language.lower()
+
     if language == "zh":
         import jieba
 
-        return jieba.cut
+        def jieba_splitter(text):
+            return list(jieba.cut(text))
+
+        return jieba_splitter
+
+    elif language == "ja":
```

Comment:
Do you mind updating the docstrings in the filters where the `get_word_splitter` is used to explain the splitting strategy we do for each language?